### PR TITLE
Fix compilation with Xcode 11.1

### PR DESCRIPTION
**Pull request by Gray-Wind**
Xcode is very strict now about files re-signing.

### CODE REVIEW

File: MonitorizareVot.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -852,8 +860,6 @@
 				27EDBD2221465D190060DB8A /* Localizable.strings in Resources */,
 				E7FEC2C51DE09E8700F2F04D /* AnswerWithTextTableViewCell.xib in Resources */,
 				28844A57233F94FD00DEEA44 /* GenericPickerViewController.xib in Resources */,
-				EF903E351DA25E3100D5BC9C /* Judete.plist in Resources */,
```
Comment (CristiHabliuc) [inline MonitorizareVot.xcodeproj/project.pbxproj]: Why is this plist removed? I know the only complaint was about the Google service info plist being referenced twice

Comment (Gray-Wind) [reply MonitorizareVot.xcodeproj/project.pbxproj]: This was removed earlier, I believe, and now Xcode deiced to get rid of obsolete link. I can drop this change, of course.

Comment (CristiHabliuc) [reply MonitorizareVot.xcodeproj/project.pbxproj]: I haven't loaded up your changes yet, I was just curious why this was removed. I also believe the counties aren't included locally anymore, but I'll double check. I'll get back to you once I test it out, thank you!

Comment (CristiHabliuc) [reply MonitorizareVot.xcodeproj/project.pbxproj]: Yes, all good, thank you!

**Pull request merged**